catalog/spatial/csw/spatial-csw-source/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/source/CswFilterDelegate.java

```diff
@@ -117,9 +117,7 @@ public FilterType and(List<FilterType> filters) {
     @Override
     public FilterType or(List<FilterType> filters) {
         areLogicalOperationsSupported();
-        if (filters.contains(Collections.singleton(null))) {
-            throw new UnsupportedOperationException("Invalid filters found in list of filters!");
-        }
+
```

Comment:
What was the finding here? Shouldn't we still check for `null` values in the `filters` list?